test/deploy/kubernetes/runtime-pods/components_server_node1.yaml

```diff
@@ -168,14 +168,14 @@ spec:
           name: "port-9443"
         - containerPort: 28049
           name: "port-28049"
-      - name: cortx-s3bgc
+      - name: cortx-s3-bgworker
```

Comment:
The format for container names should be: `<component>-<service>`
For example: `s3-bgworker`

Reply:
Lets take it as part of redefined pods infra...